e2e/executable_test.go

```diff
@@ -58,8 +58,10 @@ var _ = Describe("E2E TEST:executable", func() {
 				case "Security-Policy":
 					Expect(c.Pass).Should(BeTrue(), c.CheckName)
 				case "Signed-Releases":
+					Expect(c.Confidence).ShouldNot(Equal(10))
 					Expect(c.Pass).Should(BeFalse(), c.CheckName)
```

Comment:
Don't these 2 checks pass yet ?

Reply:
Not yet, The scores are getting better. In the past, we haven't had signed releases or signed tags.